Lib/opcode.py

```diff
@@ -142,7 +142,7 @@ def jabs_op(name, op):
 def_op('BUILD_TUPLE', 102)      # Number of tuple items
 def_op('BUILD_LIST', 103)       # Number of list items
 def_op('BUILD_SET', 104)        # Number of set items
-def_op('BUILD_MAP', 105)        # Number of dict entries (upto 255)
+def_op('BUILD_MAP', 105)        # Number of dict entries (up to 255)
```

Comment:
"(up to 255)" is outdated (in 3.6 too). Just remove it.

Reply:
elopio or whoever makes the change should refresh the patch against master.